examples/parametric/plot_compare_sfhs.py

```diff
@@ -0,0 +1,62 @@
+"""
+Compare different parametric star formation history models
+===========================
```

Comment:
Equals need to be the same length as the title. It will fail typesetting otherwise.